swagger.yaml

```diff
@@ -53,6 +53,10 @@ securityDefinitions:
     type: apiKey
     in: query
     name: image_token
+  watcherAuth:
+    type: apiKey
+    in: header
+    name: Watcher-Authorization
```

Comment:
`Authorization` is an [officially-defined](https://datatracker.ietf.org/doc/html/rfc9110#section-11.6.2) header, but when you're creating your own you would typically add an `X-` prefix.

Reply:
Looks like usage of `X-` is deprecated as per https://datatracker.ietf.org/doc/html/rfc6648

The name `Watcher-Authorization` aligns with the below recommendations -
Avoidance of X- Prefix, descriptive and unique naming, naming based on application context

This approach fits within the broader guidance from MDN’s [HTTP Headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers), where unique, non-conflicting names are emphasized.